dropshot/src/api_description.rs

```diff
@@ -1241,6 +1241,29 @@ mod test {
         );
     }
 
+    #[should_panic(expected = "route paths must begin with a '/': 'I don't \
+                               start with a slash'")]
+    #[test]
+    fn test_badpath4() {
+        #[endpoint {
+            method = PUT,
+            path = "I don't start with a slash"
+        }]
+        async fn test_twobodies_handler(
+            _: Arc<RequestContext<()>>,
+        ) -> Result<Response<Body>, HttpError> {
+            unimplemented!();
+        }
+
+        let mut api = ApiDescription::new();
+        let error = api.register(test_twobodies_handler).unwrap_err();
```

Comment:
"test_twobodies_handler" does not seem like the right name for this... how about:

```suggestion
        async fn bad_path_handler(
            _: Arc<RequestContext<()>>,
        ) -> Result<Response<Body>, HttpError> {
            unimplemented!();
        }

        let mut api = ApiDescription::new();
        let error = api.register(bad_path_handler).unwrap_err();
```

Reply:
Well that's some obvious copypasta. Thanks for the careful look.